analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ParametersCorrectOrder.CSharp11.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    public interface IInterface
+    {
```

Comment:
Discussed offline: These tests should stay.
There's an [issue](https://github.com/dotnet/runtime/issues/76673) during runtime regarding inheritance in interfaces - it's good to have tests to catch any possible regressions after it's fixed (since Roslyn might get affected).